README.md

```diff
@@ -1,7 +1,5 @@
 # Getting Started with the .NET MAUI BadgeView Control
-A quick-start project that helps you integrate the Syncfusion Badge View control with a .NET MAUI application by adding the Syncfusion MAUI Core package from NuGet. This project contains simple code to apply predefined styles, icons, and other customizations.
-
-Documentation: https://help.syncfusion.com/maui/badge-view/getting-started
+The [.NET MAUI Badge View](https://www.syncfusion.com/maui-controls/maui-badge-view?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples) is a notification control that is used to notify users of new or unread messages, notifications, or status information. This article explains the steps required to configure the .NET MAUI Badge View control for .NET MAUI Badge Notifications and customize its elements.
```

Comment:
Use as below:

The [.NET MAUI Badge View](https://www.syncfusion.com/maui-controls/maui-badge-view?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples) is a notification control that is used to notify users of new or unread messages, notifications, or status information. A quick-start project that helps you integrate the Syncfusion Badge View control with a .NET MAUI application by adding the Syncfusion MAUI Core package from NuGet. This project contains simple code to apply predefined styles, icons, and other customizations.

Reply:
Correction made